### DISCUSSION TITLE
The better prompt, the better output? What is the best option for quality & performance (hopefully less than 15s)

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2932] [Q&A] [answered]
I used this prompt: Urban Chic, Edgy, Beautiful girl, smiling, New York City, spring, blooming flowers, Central Park, iconic skyline, Empire State and Chrysler Buildings, full body shot, Tilt Blur, Shutter Speed 1/1000, F/22, White Balance, 32k, photo-realistic, Super-Resolution, intricate details, crystal clear, Pro Photo RGB, Half rear Lighting, Backlight, Dramatic Lighting, Sharp focus, Incandescent, Soft Lighting, Volumetric, Conte-Jour, Global Illumination, Screen Space Global Illumination, Scattering, Shadows, Rough, Shimmering, Lumen Reflections, Screen Space Reflections, Diffraction Grading, Chromatic Aberration, GB Displacement, Scan Lines, Ambient Occlusion, Anti-Aliasing, FKAA, TXAA, RTX, SSAO, OpenGL-Shader’s, Post Processing, Post-Production, Cell Shading, Tone Mapping, CGI, VFX, SFX, insanely detailed and intricate, hyper maximalist, elegant, dynamic pose, photography, volumetric, ultra-detailed, intricate details, super detailed, ambient, raw style, quality setting of 2, fujifilm eterna vivid 500t, cinematic lighting.

Model is Turbo Speed.
Response Time: 18s.
But the result is really poor.
![5ab35065-ee2f-4910-a782-31554b5740b2](https://github.com/lllyasviel/Fooocus/assets/91205658/f34916b4-30ae-4ca6-9320-554a4d4f59e0)

I want to the response time would be around 15, and the quality would be perfect.

How to set the configuration? Please don't hesiate to let me know.

**Comment by Rogala** [comment]
Not all models and loras work correctly in turbospeed mode, promt can be made shorter without describing all the insect life in the park.

**Reply by oldhand7** [reply]
When it came to LoRA, does fooocus use LoRA? Turbo speed use LoRA?

**Comment by wisdombox44** [comment, marked as answer]
Keep it simple. long descriptive promts confuses ai engine. Start with a simple promt, and try with disabling Fooocus default styles "Fooocus V2, Fooocus Enhance, Fooocus Sharp" try adding Fooocus V2 expansion, dont choose more then 2 styles besides fooocus V2.
Here is the simplified Promt "full body shot of a Beautiful girl, smiling, Urban Chic, Edgy, New York City, spring, blooming flowers, Central Park, iconic skyline, Empire State and Chrysler Buildings," no style selected.
![2024-05-17_21-06-51_1688](https://github.com/lllyasviel/Fooocus/assets/123548315/23108697-4299-411e-9e94-cf1fc52210e9)
![2024-05-17_21-07-04_9962](https://github.com/lllyasviel/Fooocus/assets/123548315/b78df5f2-b5d7-4b8c-9923-15630a8fe339)
![Fooocus Log 2024-05-17-1](https://github.com/lllyasviel/Fooocus/assets/123548315/1d92e13d-f88a-469c-8cca-cb4a97aaefaa)
![Fooocus Log 2024-05-17-2](https://github.com/lllyasviel/Fooocus/assets/123548315/74a80bbc-c5c9-415d-bcc7-41e8cbe74444)

**Reply by oldhand7** [reply]
Thank you for your help. Could you let me know the duration?

**Reply by oldhand7** [reply]
I want the generation within 15s hopefully with your quality. I don't know how to set model and configuration. Thanks.

**Reply by oldhand7** [reply]
![07a257c2-5446-47ea-84ba-8fdd46bc4bae](https://github.com/lllyasviel/Fooocus/assets/91205658/320ffd68-cc22-4b37-935b-5d546a5b8b5b)


I've tested the model with 768*1280 with your setting. Performance is amazing. But the quality is so poor.
Could you let me know how to set the model so as to make sure the better quality like your result?

**Comment by oldhand7** [comment]
Thank you [wisdombox44](https://github.com/wisdombox44)
![d7bd3264-ab83-417c-ad9e-b83d75f42991](https://github.com/lllyasviel/Fooocus/assets/91205658/b4035ea5-699c-4579-bb18-b67031dd23c7)
 

I tried the prompt from you but the quality of image is poorer than your result.

Appreciate it if you let me know how to set the model.

**Comment by wisdombox44** [comment]
Turbo models works best in 1024X1024 you can upscale and crop later with any photo editor
[turbo.json](https://github.com/lllyasviel/Fooocus/files/15348935/turbo.json)
here is the Turbo preset file copy this file to Fooocus>presets folder
I use Dreamshaper XL Turbo
https://civitai.com/models/112902?modelVersionId=351306
Generating times depens on your GPU I have Nvidia RTX 3060 12 gb Vram it takes 10.3 secons after first generation
<img width="844" alt="Screenshot 2024-05-17 220356" src="https://github.com/lllyasviel/Fooocus/assets/123548315/03c2fd71-e1fe-46e9-9cef-63e9099217b7">

**Comment by wisdombox44** [comment]
<img width="974" alt="Screenshot 2024-05-17 222140" src="https://github.com/lllyasviel/Fooocus/assets/123548315/db710ac2-3180-4ed4-a704-b3ec4887a369">

after you copy Turbo preset file copy this file to Fooocus>presets folder

**Reply by oldhand7** [reply]
![image](https://github.com/lllyasviel/Fooocus/assets/91205658/cfdffe67-bd53-41ca-8052-002082c7b09c)
I can't see the turbo option. Please let me know. thank you.

**Comment by wisdombox44** [comment]
here its how to do manually and tweak the settings,
https://www.youtube.com/watch?v=S4b5tQoUfBA
